### ISSUE TITLE
How to unload a project

### ISSUE THREAD

**Issue by TanvirArjel** [microsoft/vscode-dotnettools#431]
### Describe the Issue

In my solution, I have a older dotNET(4.7) class library. So In solution explorer it is showing
> The project is not supported in C# Dev Kit.

This is absolutely fine, but how can I unload this class library project so that it will be ignored during build like we do Visual Studio 2022.

### Expected Behavior

Right click on the project should show `Unload Project` option.

### Environment Information

- Mac OS
- C# Dev Kit - v0.4.6

**Comment by lifengl** [comment]
C# Dev Kit doesn't build solution like Visual Studio does. It basically runs CLI command dotnet build, which is independent to the solution explorer view, so unloading the project is not very meaningful in this context.

**Comment by ADIX7** [comment]
It is actually. I frequently unload test project(s) when I work with something in the main project(s) that has a lot of reference in the test project(s). So basically first I understand/refactor the program code and after that I reload the test project(s) / fix them.

**Comment by SHiLLySiT** [comment]
+1 for adding support to unload projects.

I recently got put on a very large Unity project that has 134 projects in the solution. Loading this project in VS Code requires 10-12 gigs of RAM, which I don't have to spare. 

I've had to switch to regular VS since it supports loading specific projects 😢

**Comment by ajay-mundi** [comment]
+1 yes this would be very useful for me

**Comment by huzaifacontour** [comment]
+1 really need this

**Comment by ArtemVolsh** [comment]
+1 here

**Comment by Rajan-Aggarwal** [comment]
This would be very useful to me as well. Besides, the entire point of VSCode is to be able to use a software that's less heavy that Visual Studio. Loading every project would defeat that purpose.

**Comment by DennisDyallo** [comment]
Yes, I want this. We have a bunch of projects that I dont need to have builds triggered when I'm hitting Ctrl Shift B. E.g. Sample code projects, integrationtests, unit tests. Disabling these projects in Visual Studio or Rider will substantially improve build times for me.

**Comment by MiffOttah** [comment]
I'm going to want to concur on this. I have a solution that was developed with cross-platform support in mind that has central class libraries containing the business logic and a Windows-specific frontend. I want to port it to Linux by writing a new frontend, but that's not possible due to the the net7.0-windows target failing to build. Unloading the project would be a straightforward solution since the project is not relevant under Linux.
